joomla/joomla-cms
# [5.2] [Guided tours] Add missing auto start parameter in the user interface of tours

### Summary of Changes

The autostart parameter has been introduced in Joomla 5.1 to facilitate the autostart feature and mainly the welcome tour.

This PR goes a step further and adds the ability to make a tour auto-start-able in the interface.
The purpose of the PR is to just add the ability to modify the parameter from the edit view.
The functionality for auto-starting tours is presented in a separate PR #43814.

![image](https://github.com/joomla/joomla-cms/assets/5964177/4b58563b-c11e-4229-ad5f-b1fd92648bc8)

### Testing Instructions

Go to System -> Guided Tours.
Pick a tour.
Check that the option is present.
Set it on.
Duplicate the tour.
Make sure the duplicated tour replicates the original tour auto-start value.

Note. The welcome tour, at present (until the addition of the joint PR), if set to auto-start, will run once and be automatically set to auto-start: false after the run.
Tours set to auto start will not start automatically with this PR alone.

### Actual result BEFORE applying this Pull Request

No auto start parameter.

### Expected result AFTER applying this Pull Request

The auto start parameter can be changed from the tour edit view.

### Link to documentations
Please select:
- [x ] Documentation link for docs.joomla.org: [Edit Tour](https://docs.joomla.org/Help5.x:Guided_Tours:_New_or_Edit_Tour)
